data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
> @tresf The CSS now gives great flexibility with regards to the colors. So if the user decided to make the patterns white it would be wise to take a different color for the text background or the text color itself

I'm talking about testing the built-in functionality.  This has nothing to do with custom themes or custom CSS, this has to do with testing this feature against the color capabilities of our BB patterns.